publisher/socials/twitter.py

```
@@ -0,0 +1,121 @@
+    ])
+    ckeditor_config['allowedContent'] = (
+        'img[!src,alt,width,height];'               # изображения
+        'div[*]{*}(tweet-*); span[*]{*}(tweet-*);'  # твиты
+        'a[!href];'                                 # твиттер-ссылки
+    )
+    ckeditor_config['extraPlugins'] += ',tweet_splitter'
+    
+    def clean_publish_data(self, content):
+        
+        # 1. Разбиваем на твиты
+        tweets = re.split(r'<div class="tweet-separator">.*?</div>', html, flags=re.DOTALL)
```

Ruff (0.12.2)

Undefined name `re`

(F821)